taskcluster/async/asyncclient.py

```diff
@@ -50,10 +53,16 @@ class AsyncBaseClient(BaseClient):
     help with this.
     """
 
+    def __init__(self, *args, **kwargs):
+        super(AsyncBaseClient, self).__init__(*args, **kwargs)
+        self._implicit = False
+        if self.session is None:
+            self._implicit = True
+
     def _createSession(self):
         """ Create a new aiohttp session.
         """
-        return createSession()
+        return None
```

Comment:
A comment explaining why this function is stubbed for async would be useful, doesn't have to be in the docstring or anything too detailed.

Reply:
Done.